src/provisioner/cluster.py

```diff
@@ -86,12 +87,24 @@ def _get_kvs(self, prefix, node):
             kvs.append((prefix, node))
         return kvs
 
+    @staticmethod
+    def _add_comp_version(components):
+        """Add rpm version key for each component."""
+        updates_list = []
+        cortx_release_info = CortxReleaseInfo()
+        for comp in components:
```

Comment:
Lets use full variable name:
`for component in components`

Reply:
Done